README.md

```diff
@@ -2,78 +2,65 @@
 
 
 # 🚀 Quick start
+1.  **Prerequisites**
 
-1.  **Start developing.**
+    * git
+    * [nvm](https://github.com/creationix/nvm)
+
+1.  **Get up and running**
 
     Navigate into your new site’s directory and start it up.
 
     ```sh
-    
-    cd my-default-starter/
-    gatsby develop
+    git clone git@github.com:neontribe/www.git
+    cd www
+    nvm use
+    npm install -g yarn
```

Comment:
So this will install the given bundle globally? I might be misunderstanding but would we not want it to be limited to this project?

Reply:
It'll install it semi-globally... That is, it'll install it in the currently activated nvm-managed node environment. It won't affect any other node install, nor will it require any elevated privileges.

Reply:
Ah ok.